plugins/catalog-backend/src/database/CommonDatabase.ts

```diff
@@ -207,13 +210,13 @@ export class CommonDatabase implements Database {
 
   async entities(
     txOpaque: Transaction,
-    filter?: EntityFilter,
-  ): Promise<DbEntityResponse[]> {
+    request?: DbEntitiesRequest,
+  ): Promise<DbEntitiesResponse> {
     const tx = txOpaque as Knex.Transaction;
 
     let entitiesQuery = tx<DbEntitiesRow>('entities');
```

Comment:
I would have thought this would use the entities_search table.  Am I missing something?

Reply:
Oh but it does - look inside the query a couple lines farther down. There's a comment mentioning the use of IN rather than JOIN too

Reply:
The filtering as such was actually not changed by this PR.

Reply:
Right, mobile GH app got me. Should have known.